src/components/AddBook.js

```diff
@@ -1,24 +1,51 @@
+/* eslint-disable no-unused-vars */
```

Comment:
- [ ] Please kindy resolve the issue that this linter is pointing at, the ESLint rule `no-unused-vars` is used to flag variables that are declared but not used in your code. It's generally a good practice to keep your code clean and remove any unused variables. 👍